crates/anonymization/src/core/noise.rs

```diff
@@ -81,12 +82,15 @@ where
     }
 }
 
+// Future suggestion: make this structure generic over a trait implemented by
+// the distribution
```

Comment:
Maybe use an issue to track this. Comments tends to get forgotten, even with a `TODO` marker. And this is cleaner.